ChangeLog.d/remove_obsolete_tls_features.txt

```diff
@@ -4,3 +4,4 @@ API changes
    * Drop support for compatibility with our own previous buggy implementation of truncated HMAC (MBEDTLS_SSL_TRUNCATED_HMAC_COMPAT).
    * Drop support for TLS record-level compression (MBEDTLS_ZLIB_SUPPORT).
    * Drop support for RC4 TLS ciphersuites.
+   * Drop single-DES ciphersuits.
```

Comment:
Typo: ciphersuitEs. While at it, please align the wording on the previous entry: "Drop support for single-DES TLS ciphersuites."